### PR TITLE
Tools and resources updates an updating task pages to use the tool format - issue #129

### PR DESCRIPTION
**Pull request by shoaibsufi**
This is the pull request for issue https://github.com/EVERSE-ResearchSoftware/RSQKit/issues/129 - this is a draft pull request at the moment as it's a slightly time consuming branch I wanted to give sight of the work being done here - once it is complete I will change the pull request to a proper one.

### CODE REVIEW

File: _data/tool_and_resource_list.yml
Comment (dgarijo) [inline _data/tool_and_resource_list.yml:9]: I suggest to have a URL here for now.
```suggestion
#   registry: This is a list of registries where the tool is available 
```

File: _data/tool_and_resource_list.yml
Comment (dgarijo) [inline _data/tool_and_resource_list.yml:12]:
```suggestion
#   qualityDimension: Research Software Quality Dimension identifier (see dimensions.yaml in this folder) of the quality dimension associated with this tool. More than one dimension may apply (in a list)
#   qualityIndicator:  Research Software Quality indicator (see indicators.yaml in this folder) associated with this tool. More than one quality indicator may be declared per tool (in a list)
```

File: _data/tool_and_resource_list.yml
Comment (dgarijo) [inline _data/tool_and_resource_list.yml:1]:
```suggestion
# List of tools and resources
# Conforms to https://w3id.org/everse/rs/0.0.1
```
I will publish the metadata schema in https://w3id.org/everse/rs/0.0.1 (soon)

**Comment by dgarijo** [comment]
@shoaibsufi please see if my suggestions are acceptable. This is in line with what was discussed today, in order to address @vuillaut comments. If you incorporate them, I will update the tools with dimensions/indicators that I am uploading as a separate PR.

File: _data/tool_and_resource_list.yml
Comment (shoaibsufi) [reply _data/tool_and_resource_list.yml:1]: ok - will wait for it to be published before I merge

File: _data/tool_and_resource_list.yml
Comment (shoaibsufi) [reply _data/tool_and_resource_list.yml:9]: No - this is literally the list of 'registry:seach-string' combinations - which are searched to further information about this tool entry - I will make it clearer but not adding this suggestion.

File: _data/tool_and_resource_list.yml
Comment (shoaibsufi) [reply _data/tool_and_resource_list.yml:12]: I don't like camel case  - and for it to be CamelCase it would be QualityDimension - I think it's much more readable if we use lower_case_with_underscores so quality_dimension and quality_indicator

Comment (dgarijo) [reply _data/tool_and_resource_list.yml:12]: schema.org uses camelCase notation, and CodeMeta as well (https://codemeta.github.io/terms/), plus half of semantic web vocabularies. The schema we are proposing is aligned with them. Adding a different tag for rsqkit makes things more difficult to maintain in my opinion. 
qualityDimension is correct camel case notation https://en.wikipedia.org/wiki/Camel_case

File: _data/tool_and_resource_list.yml
Comment (dgarijo) [reply _data/tool_and_resource_list.yml:9]: oh, I misunderstood then

File: _data/tool_and_resource_list.yml
Comment (dgarijo) [reply _data/tool_and_resource_list.yml:1]: Please go ahead an publish, I will make it available soon

**Review by anenadic — APPROVED** [review]

**Pull request merged**
